Automattic/jetpack
# add promoted posts block after publishing content

#### Changes proposed in this Pull Request:

**This is a work in progress**

This adds a new Post-publish panel in the block editor.

**More details to come soon**

#### Other information:


#### Jetpack product discussion

- pedfti-DJ-p2#comment-849
- p1HpG7-jCm-p2
- Related PR: #27906

#### Does this pull request change what data or activity we track or use?

* TBD, this will depend on the implementation

#### Testing instructions:

* Start on a Jetpack site that's connected to WordPress.com.
* Switch to this branch, or use the Jetpack Beta plugin to run this branch.
* Ensure Beta blocks are enabled on your site. If you test on a WoA site, for example, add the following: `add_filter( 'jetpack_blocks_variation', function() { return 'beta'; } );`. If you work locally, you can add 
`define( 'JETPACK_BETA_BLOCKS', true );` in your `docker/wordpress/wp-config.php`.
* Go to Posts > Add New
* In your console, check that the new extension is now available: `'post-publish-promote-post-panel' in Jetpack_Editor_Initial_State.available_blocks`
* Publish a post.
* Check the post-publish sidebar panel and click on the button.
